app/models/container_group.rb

```diff
@@ -85,8 +85,6 @@ def disconnect_inv
     _log.info "Disconnecting Pod [#{name}] id [#{id}] from EMS [#{ext_management_system.name}]" \
     "id [#{ext_management_system.id}] "
     self.container_definitions.each(&:disconnect_inv)
-    self.old_ems_id = ems_id
-    self.ext_management_system = nil
```

Comment:
Talked with @zakiva, we're also clearing `self.container_node_id = nil` which will make less sense after https://github.com/ManageIQ/manageiq-schema/pull/22 starts archiving nodes.
I think let's get both this and https://github.com/ManageIQ/manageiq-schema/pull/22 merged, then revisit these.